### DISCUSSION TITLE
saf-training/courses/user/12

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#33] [Ideas]
# saf-training/courses/user/12

The MITRE Security Automation Framework Team's Introduction to InSpec profile development

https://mitre.github.io/saf-training/courses/user/12.html

**Comment by grouse142** [comment]
Link in 12.2 fails - 404 error code

**Reply by grouse142** [reply]
12.4: write the full command "saf attest create -h" so it works on the first try in the CLI

**Comment by karikarshivani** [comment]
12.5 has a block that has the command as well as the output - need to change it to code tabs

**Comment by p-oneil** [comment]
Moved to PR

**Comment by unfortuNateLee** [comment]
saf CLI asks for "control ID" - this is confusing, since a) the check ID / requirement ID / whatever ID that it's asking for is NOT an 800-53 control ID (but those ARE listed in the Heimdall output). Suggest that you use a more specific (and less overloaded) term.

**Comment by unfortuNateLee** [comment]
Suggest that you show more comparison columns in Comparison mode - it's non-intuitive that the charts above the comparison show ALL of the loaded files but the comparison area only shows 2 at a time. Pick a reasonable max number >2 (maybe 4?) or (better) make it dependent on the screen width.
